src/access/AccessControl.cpp

```diff
@@ -171,6 +171,23 @@ char GetPrivilegeStringForLogging(Privilege privilege)
     return 'u';
 }
 
+char GetActionStringForLogging(MsgType action)
```

Comment:
please fix return type or name: char is not a string.

Reply:
I see the previous implementation also did that :( So pre-existing.... sigh. Can be a followup that probably never happens.

Reply:
The intent was to log a single character, not a string.